### PR TITLE
[1LP][RFR] CB: change in flash msg

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
Fix flash

{{pytest: cfme/tests/automate/custom_button -k "test_custom_button_automate" -v}}

### CODE REVIEW

**Comment by digitronik** [comment]
service test fails on 5.10. not repleted to changes. It will fix in #8625

File: cfme/tests/automate/custom_button/test_cloud_objects.py [hunk 1]
```diff
@@ -294,7 +294,12 @@ def test_custom_button_automate(appliance, request, submit, setup_objs, button_g
                 'echo -n "" > ' "/var/www/miq/vmdb/log/automation.log"
             )
             custom_button_group.item_select(button.text)
-            view.flash.assert_message('"{}" was executed'.format(button.text))
+
+            view.flash.assert_message(
+                '"{btn}" was {k}'.format(
+                    btn=button.text, k="executed" if appliance.version < "5.10" else "launched"
```
Comment (izapolsk) [inline cfme/tests/automate/custom_button/test_cloud_objects.py]: please don't use if when it doesn't fit one row. it isn't well readable.

**Comment by izapolsk** [comment]
@digitronik, PRT failed against 5.10

**Comment by digitronik** [comment]
@izapolsk PRT failures not related with flash. I am need to check `StaleElementReferenceException`. Please have a glance. I added flash msg changes.

**Comment by lina-nikiforova** [comment]
Let's keep PRs small and move the flash message change further. I think it's better to open a separate PR for a new issue.

**Comment by izapolsk** [comment]
@digitronik, PRT results don't shed a light on what's wrong there. please remind me later, I'll look closer locally at the issue.

**Pull request merged**